e2e/pages/caseView/caseView.js

```diff
@@ -22,4 +22,8 @@ module.exports = {
   selectTab(tab) {
     I.click(tab);
   },
+
+  goToCaseList(){
+    I.click('Case List');
```

Comment:
would be nice if you could put a locator in here, that way we know we will always click the right thing

Reply:
👍